03.bowling/bowling.rb

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = []
+shots.each_slice(2).each_with_index do |s, i|
+  frames << [s, i]
+end
```

Comment:
```suggestion
frames = shots.each_slice(2).each_with_index.to_a
```

でよさそうです。
https://docs.ruby-lang.org/ja/latest/method/Enumerable/i/entries.html

Reply:
ありがとうございます！
最初に空配列を定義しなくても`to_a`メソッドで配列として代入したらよいということですね。
訂正してプッシュします。